src/sources/mongoose/resources/hooks/update-password.ts

```diff
@@ -0,0 +1,35 @@
+// eslint-disable-next-line max-len
+import { ActionContext, ActionRequest, Before } from 'adminjs';
+import argon2 from 'argon2';
+import {isEditAction, isNewAction, isPOSTMethod} from "../../../../admin/admin.utils";
+
+export const updatePassword = async (
```

Comment:
we have a password feature created by Wojtek: https://github.com/SoftwareBrothers/adminjs-passwords

might be better to use that as people will most likely find it and search for examples